diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/EmptyBlockWarnTest.kt

```diff
@@ -0,0 +1,51 @@
+package org.cqfn.diktat.ruleset.chapter3
+
+import com.pinterest.ktlint.core.LintError
+import org.cqfn.diktat.common.config.rules.RulesConfig
+import org.cqfn.diktat.ruleset.constants.Warnings
+import org.cqfn.diktat.ruleset.rules.DIKTAT_RULE_SET_ID
+import org.cqfn.diktat.ruleset.rules.EmptyBlock
+import org.cqfn.diktat.util.lintMethod
+import org.junit.Test
+
+class EmptyBlockWarnTest {
```

Comment:
Is there any reason why tests on this functionality are present in two classes, `EmptyBlockWarnTest` and `BlockStructureBracesWarnTest `?

Reply:
Sorry, didn't notice that you have added tests for two rules.